src/signals/incident/components/form/MapSelectors/Asset/Selector/WfsLayer/__tests__/WfsLayer.test.tsx

```diff
@@ -39,7 +38,7 @@ const endpoint = 'https://endpoint/?version=2'
 const promise = Promise.resolve()
 const assetSelectProviderValue: AssetSelectValue = {
   selection: [],
-  location: new LatLng(0, 0),
+  location: [0, -0],
   meta: {
```

Comment:
why do you use a negative 0 here?

Reply:
Hm, must've happened when I reformatted the whole file. Definitely wasn't intentional. Doesn't make a difference, tho.